azurerm/internal/services/netapp/tests/resource_arm_netapp_snapshot_test.go

```diff
@@ -204,6 +219,30 @@ resource "azurerm_netapp_snapshot" "test" {
   volume_name         = azurerm_netapp_volume.test.name
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
+
+  tags = {
+    "foo" = "bar"
```

Comment:
Could we ensure capitals work
```suggestion
    "FoO" = "BaR"
```